src/App.jsx

```diff
@@ -1,22 +1,25 @@
-import { BootstrapDartboard as FindAddress } from '@ugrc/dart-board';
-import Sherlock, { WebApiProvider } from '@ugrc/sherlock';
+import {
+  Drawer,
```

Comment:
Should this component be moved into Atlas since it's dependent on the app structure?

Reply:
i'm not sure yet.